### ISSUE TITLE
Remote file copy/paste functionality on local Windows 11 causes apparent constant IO, spinning blue circle on mouse cursor

### ISSUE THREAD

**Issue by liquiddandruff** [rustdesk/rustdesk#4160]
### Bug Description

After toggling "Allow file copy and paste" and copying a large 2GB file from remote to local (which took a while even though the machine was on LAN, I'll leave that for another issue), the local machine started having weird "constant io" behaviour indicated by the blue spinning circle indicator above the mouse cursor.

This circle spinning behaviour permanently persists even after the file copy operation has completed, and in fact even after disconnecting from the remote machine. Only after killing the main RustDesk process does this behaviour stop. 

The blue circle indicator flashes briefly for ~0.1 sec, goes away for 1 sec, then repeats.

### How to Reproduce

Not sure exact steps, but I was copying a bunch of files from remote system to local system. Relevant files copied were 1 large 2GB file, then a bunch of files and subdirectories not larger than ~50MB in total, but there were a lot of files (git repos, node repos, etc),

### Expected Behavior

There should not be apparent persistent IO on the local machine showing up as a blue circle indicator on the cursor every 1 sec, even after disconnecting from the remote system.

### Operating system(s) on local side and remote side

Windows 11 -> Windows 10

### RustDesk Version(s) on local side and remote side

1.1.9

**Comment by liquiddandruff** [comment]
It just happened again and I didn't even need to initiate any file operation this time. The blue spinning indicator appeared right after I connected to the remote machine (the `Allow file copy and paste` was already ticked, remembered from my last session).

After disconnecting from the session, the blue indicator flashing persists. 

Only when I exit the main rustdesk process, the blue indicator stops flashing.

**Comment by rustdesk** [comment]
#2283

**Comment by liquiddandruff** [comment]
👍 though in my case file copying works, just there's the IO indicator after session termination. But I suppose this will be fixed once file copying in general is more polished. Thanks!

**Comment by rustdesk** [comment]
@fufesou FYI

**Comment by fufesou** [comment]
@R-ohit-B-isht FYI
